packages/azure-container-services/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@instana/azure-container-services",
+  "version": "2.35.0",
+  "description": "Instana tracing and monitoring for Node.js based Azure Container Services",
+  "author": {
+    "name": "Arya Mohanan",
+    "email": "arya.mohanan@ibm.com"
+  },
+  "homepage": "https://github.com/instana/nodejs/blob/main/packages/azure-container-services/README.md",
+  "main": "src/index.js",
+  "files": [
+    "src",
+    "esm-loader.mjs"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
+    "lib": "src"
```

Comment:
Question: What is this?

Reply:
Haha, I think this is a general standard for specifying the structure of our packages; I saw it in all our other packages as well. I can remove this even if it is not a necessary field.

Reply:
Why `lib`? We do not have a lib folder.

Reply:
https://areknawo.com/whats-what-package-json-cheatsheet/

> directories may be one of these mysterious fields that not many know what it exactly does. 🤔 I'll tell you - it's mostly just meta-info. Exactly two fields provide some functionality and nothing else. Of course, all of them should have a form of string pointing to different directories.

> lib - meta info about where your library is exactly located in your package;

I didn't know that. Okay let's keep :)

Reply:
Ah, no, nevermind, I confused this with the `files` field, my bad 🤦

Reply:
will keep this.